### PR TITLE
Added picture support

### PR DESCRIPTION
**Pull request by jacobj10**

### CODE REVIEW

**Comment by illegalprime** [comment]
That looks great! But the images are not always square and when people don't have a social link it shows a dash sign. Maybe it just shouldn't show anything if they don't have links? And I think fitting the images to the whole area is a css fix.

File: _pages/about.html [hunk 1]
```diff
@@ -8,11 +8,95 @@
...
+                {% assign mod = num_members | modulo:per_row %}
+                {% if mod == 0 %}
+                 <div class="row">
+                {% endif %}
+                <div class="col col-lg-4 col-md-4 col-sm-12">
+                    <center>
+                    <div class="hovereffect">
+                        <center>
+                            <div class="crop">
+                            <img class="img-responsive" src="/static/images/members/{{ member.name }}" alt="" width="300" height="300">
```
Comment (illegalprime) [inline _pages/about.html]: Use a background div image so you can have it zoom-fill: https://stackoverflow.com/a/11763434

File: _pages/about.html [hunk 2]
```diff
@@ -8,11 +8,95 @@
...
+                            <div class="rotate">
+                                <p class="group1">
+                                    {% if member.link1 %}
+                                        <a href= {{ member.link1 }}  >
+                                            <i class="fa fa-{{ member.media1 }}"></i>
+                                        </a>
+                                    
+                                    {% else %}
+                                        <a href= "#">
+                                            <i class="fa fa-minus"></i>
```
Comment (illegalprime) [inline _pages/about.html:39]: Do not show anything if they don't have it.

**Review by illegalprime — CHANGES_REQUESTED** [review]
Beautiful!

**Comment by jacobj10** [comment]
all right, should be fixed!

**Review by illegalprime — APPROVED** [review]

**Pull request merged**